### PR TITLE
Commune model

### PR DESCRIPTION
**Pull request by RonTuretzky**
#24

### CODE REVIEW

**Review by FaezZangariya — APPROVED** [review]
just fix the flake8 errors , other than that looks good to me for now

File: commune_app/models.py [hunk 1]
```diff
@@ -1,3 +1,12 @@
-# from django.db import models
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.contrib.postgres.fields import ArrayField
 
-# Create your models here.
+
+class Commune(Models.Model):
```
Comment (bennypowers) [inline commune_app/models.py:6]: we'd prefer to have docstrings on all classes

**Review by bennypowers — APPROVED** [review]

File: commune_app/models.py [hunk 2]
```diff
@@ -1,3 +1,51 @@
...
+from django.utils import timezone
+from django.contrib.auth.models import AbstractUser
+from django.core.validators import RegexValidator
 
-# Create your models here.
+# regex to filter valid phone numbers
+phone_regex = RegexValidator(regex=r'^\+?1?\d{9,15}$', message="Phone number format: '+999999999'")
+
+
+class User(AbstractUser):
```
Comment (kobihk) [inline commune_app/models.py:10]: This code already merged as part of:
https://github.com/redhat-beyond/commune/pull/27

please rebase

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -67,3 +67,12 @@ Must be a list of measurable achievements that indicate the work in the issue is
 4. Don't just hope for the code to work, Check it yourself!
 5. Be strict about temporary code
 6. Visualize the bigger picture
+
```
Comment (kobihk) [inline CONTRIBUTING.md:70]: It is already part of:
https://github.com/redhat-beyond/commune/blob/main/README.md 
Please remove it

**Review by kobihk — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (kobihk) [reply CONTRIBUTING.md:70]: @RonTuretzky please reply to the comments and rebase

**Pull request closed without merging**